lib/post_to_slack.rb

```diff
@@ -3,8 +3,14 @@ def self.post_slack_msg( channel, message, attachments )
     token = ENV["BOT_AUTH"]
     begin
       attachments = attachments.to_json
-      options = { query: { channel: channel, text: message, attachments: attachments }, headers: { 'Authorization' => "Bearer #{token}"} }
-      response = HTTParty.post('https://slack.com/api/chat.postMessage', options)
+      if channel==="RESULTS"
+        options = { query: { text: message, attachments: attachments } }
+        url = ENV["RESULTS_CHANNEL"]
+      else
+        options = { query: { channel: channel, text: message, attachments: attachments }, headers: { 'Authorization' => "Bearer #{token}"} }
+        url = 'https://slack.com/api/chat.postMessage'
+      end
+      response = HTTParty.post(url, options)
```

Comment:
Why not use the ruby slack gem? https://github.com/slack-ruby/slack-ruby-client

Reply:
Feel like I'll have to rewrite a lot of my app now to switch to it. 😳 Looking into that can maybe be a later to-do...for now I wanted to work with Slack's API directly.